internal/provider/resource_target.go

```diff
@@ -102,6 +104,12 @@ func resourceTarget() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			targetAddressKey: {
+				Description:   "Optionally, a valid network address to connect to for this target. Cannot be used alongside host sources.",
```

Comment:
```suggestion
				Description:   "Optionally, a valid network address to connect to for this target. Cannot be used alongside host_source_ids.",
```
Or optionally you could say host sources (host_source_ids). I just want to specifically call out the field it cannot be used alongside.

Reply:
Thanks for reviewing and pointing out the docs @louisruch, added them with [9c4f84f](https://github.com/hashicorp/terraform-provider-boundary/pull/308/commits/9c4f84f43cc029250919df5d790cc28aa89630d7) 😄